pkg/operator/cluster/controller.go

```diff
@@ -139,15 +139,15 @@ func (c *ClusterController) onAdd(obj interface{}) {
 	}
 
 	if cluster.Spec.MonCount <= 0 {
-		logger.Warningf("mon count is 0 or less (given: %d), needs to be greater 0", cluster.Spec.MonCount)
+		logger.Warningf("mon count is 0 or less (given: %d), should be greater than 0, defaulting to %d", cluster.Spec.MonCount, defaultMonCount)
```

Comment:
should we change this to info?

Reply:
@travisn The intent of the message is to be a warning for the user.